query/src/servers/http/v1/multipart_format.rs

```diff
@@ -128,6 +131,7 @@ impl MultipartWorker {
 impl MultipartFormat {
     pub fn input_sources(
         name: &str,
+        ctx: Arc<QueryContext>,
```

Comment:
It would be better to put the `ctx` as the first argument, we can refine it in another PR.